src/plugins/intel_gpu/tests/functional/shared_tests_instances/single_layer_tests/multinomial.cpp

```diff
@@ -12,14 +97,26 @@ const std::vector<ov::test::ElementType> netPrecisions = {
     ov::element::f32,
 };
```

Comment:
`netPrecisions` is not used, please del it
Could you please add tests for f16 data type as well?